modules/mod_video_embed/mod_video_embed.erl

```diff
@@ -312,26 +312,32 @@ event(#submit{message={add_video_embed, EventProps}}, Context) ->
                 {content_group_id, ContentGroupdId}
             ],
 
-            case m_rsc:insert(Props, Context) of
-                {ok, MediaId} ->
-                    spawn_preview_create(MediaId, Props, Context),
-
-                    {_, ContextLink} = mod_admin:do_link(z_convert:to_integer(SubjectId), Predicate,
-                                                         MediaId, Callback, Context),
-
-                    ContextRedirect = case SubjectId of
-                        undefined ->
-                            case Stay of
-                                false -> z_render:wire({redirect, [{dispatch, "admin_edit_rsc"}, {id, MediaId}]}, ContextLink);
-                                true -> ContextLink
-                            end;
-                        _ -> ContextLink
-                    end,
-                    z_render:wire([
-                        {dialog_close, []},
-                        {growl, [{text, "Made the media page."}]}
-                        | Actions], ContextRedirect);
-                {error, _} = Error ->
+            try
+                {ok, MediaId} = m_rsc:insert(Props, Context),
+                spawn_preview_create(MediaId, Props, Context),
+
+                {_, ContextLink} = mod_admin:do_link(z_convert:to_integer(SubjectId), Predicate,
+                                                     MediaId, Callback, Context),
+
+                ContextRedirect = case SubjectId of
+                    undefined ->
+                        case Stay of
+                            false -> z_render:wire(
+                                {redirect, [
+                                    {dispatch, "admin_edit_rsc"}, {id, MediaId}
+                                ]},
+                                ContextLink
+                            );
+                            true -> ContextLink
+                        end;
+                    _ -> ContextLink
+                end,
+                z_render:wire([
+                    {dialog_close, []},
+                    {growl, [{text, "Made the media page."}]}
+                    | Actions], ContextRedirect)
+            catch
+                {error, Error} ->
                     lager:error("[mod_video_embed] Error in add_video_embed: ~p on ~p", [Error, Props]),
```

Comment:
Line 341 is too long:                     lager:error("[mod_video_embed] Error in add_video_embed: ~p on ~p", [Error, Props]),.